### PR TITLE
exec/execAdmin with execCC fix

### PR DESCRIPTION
**Pull request by Satty9361**
made these changes so that an exec/execAdmin following an execCC runs in the new channel. 

Have not incorporated channel update for dm templates ( eg: punishment dm templates) to prevent loops (eg: warn -> warn dm execCC -> execAdmin warn ...) .

### CODE REVIEW

File: commands/tmplexec.go [hunk 1]
```diff
@@ -77,7 +77,13 @@ type cmdExecFunc func(cmd string, args ...interface{}) (interface{}, error)
 // Returns 2 functions to execute commands in user or bot context with limited about of commands executed
 func TmplExecCmdFuncs(ctx *templates.Context, maxExec int, dryRun bool) (userCtxCommandExec cmdExecFunc, botCtxCommandExec cmdExecFunc) {
 	execUser := func(cmd string, args ...interface{}) (interface{}, error) {
-		mc := &discordgo.MessageCreate{ctx.Msg}
+		messageCopy := *ctx.Msg
+		if ctx.Msg.ChannelID != ctx.Msg.GuildID { //punishment dms shouldnt be able to execCC followed by exec/execAdmin to prevent loops
```
Comment (jonas747) [inline commands/tmplexec.go]: Don't think this is needed, as the moderation commands only returns when the punish dm has been sent, can there can only be 1 instance of each command per user being executed

Comment (Satty9361) [reply commands/tmplexec.go]: Ah okay , removed the check, it is still necessary to check if CS is a nil pointer before accessing CS.ID so kept that check. It is nil pointer in some cases like direct exec/execAdmin within punishment dms ( i got a runtime panic when i was running without this condition earlier ).

**Comment by Satty9361** [comment]
Do you need any other changes @jonas747  , I had made the changes you had suggested last time.

**Comment by jonas747** [comment]
hmm any reason why it's only for execUser and not execAdmin?

**Comment by Satty9361** [comment]
I have added for both , execUser as well as execBot( execAdmin).

**Comment by jonas747** [comment]
ah my bad didn't see that

**Pull request merged**